plugins/LilyPad/Tooltips.cpp

```diff
@@ -35,8 +35,8 @@ LPWSTR dialog_message(int ID, bool *updateText)
                    L"Note 2: This does not enable the mouse to function as an in-game mouse in PS2 games that support a USB mouse or lightgun."
                    L"This requires a USB plugin, while LilyPad is a PAD plugin(PAD means it emulates devices that are to be plugged into the PlayStation controller port that connects the DualShock 2 controller).";
         case IDC_MOUSE_UNFOCUS:
-            return L"Starts the emulation without the mouse being in focus by the application.\n\n"
-                   L"Without mouse focus, the mouse can still be used outside of the emulation.\nWith the mouse in focus it can only be used as an input for the emulator.";
+            return L"Enabled: Mouse is unfocused and can be used with emulation and outside it for general usage.\n\n"
```

Comment:
Possibly a better way to put this? like "Enabled: Mouse is unfocused and can be used in emulation and generally inside your OS.\n\n"